subhub/db.py

```diff
@@ -117,7 +117,7 @@ def mark_deleted(self, uid: str) -> bool:
             return False
 
 
-def _create_hub_model(table_name_, region_, host_):
+def _create_hub_model(table_name_, region_, host_) -> Any:
```

Comment:
`_create_hub_model` returns an optional model possibly?  I can get that this would error out so optional makes sense.

Reply:
mypy seems to be unable to distinguish the HubEventModel in the return from what is actually returned and something like `@47` (random number each time) so it then fails, it passes with Any.